### PR TITLE
VAGOV-5287: migrate from PR

### PR DESCRIPTION
**Pull request by beeyayjay**
To test:
1. Go to /admin/structure/migrate/manage/va_tests/migrations/va_benefits_burials/execute and execute migration.
2. Go to /section/burials-and-memorials-benefits-hub. There should be 12 nodes corresponding to Burials & Memorials detail pages.

### CODE REVIEW

File: docroot/modules/custom/va_gov_migrate/config/install/migrate_plus.migration.va_support_service.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ source:
   plugin: support_service
   urls:     # This can be a list urls to crawl or .md files to read.
     - "/disability/index.md"
-    - "/education/index.md"
+#    - "/education/index.md" - Overwrites "Ask a question online" with education-specific link
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_migrate/config/install/migrate_plus.migration.va_support_service.yml]: Was this comment supposed to be for the whole line, or just the comment at the end?   If the whole line, why keep the line?

File: docroot/modules/custom/va_gov_migrate/src/Plugin/migrate/source/MetalsmithSource.php [hunk 2]
```diff
@@ -82,19 +96,23 @@ public function __construct(array $configuration, $plugin_id, $plugin_definition
...
-        if (!empty($contents)) {
-          $file_info = json_decode($contents);
-          $this->addRow($file_info->git_url, $url);
+    if (!empty($this->prNumber)) {
+      $this->readPr();
+    }
+    else {
+      foreach ($this->urls as $url) {
+        // If it's a markdown file, process it.
+        if (substr($url, -3) == '.md') {
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_migrate/src/Plugin/migrate/source/MetalsmithSource.php]: Minor: Is there a reason that the stronger pathinfo() function is not used to get the file extension?  It has some safeties built in for obtaining file extensions.
```
$ext = pathinfo($url, PATHINFO_EXTENSION);
if (strtolower($ext) === 'md') {
```

File: docroot/modules/custom/va_gov_migrate/config/install/migrate_plus.migration.va_support_service.yml (hunk 1 above)
Comment (beeyayjay) [reply docroot/modules/custom/va_gov_migrate/config/install/migrate_plus.migration.va_support_service.yml]: Good point. Removed the line

**Comment by swirtSJW** [comment]
rebuild

**Comment by swirtSJW** [comment]
@jonpugh Is this behat failure something that is still being resolved?

**Comment by swirtSJW** [comment]
@jonpugh If this behat test issue can't get resolved soon, will you remove your commits to this branch so that we can proceed with getting Beths' work into the release today?

**Comment by swirtSJW** [comment]
@jonpugh I removed your  behat commits from this PR.

**Pull request merged**